CONTRIBUTING.md

```diff
@@ -7,23 +7,211 @@ repository before making a change.
+      "**/bower_components": true,
+      "**/*.code-search": true
+    },
+    // ts gives me the power to not rely on word matching
+    "editor.wordBasedSuggestions": false,
+    "gitlens.hovers.enabled": false,
+    "editor.codeActionsOnSave": {
+      "source.fixAll.eslint": true
+    },
+    "[javascript]": {
+      "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+    },
+    "[typescript]": {
+      "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": false
+      }
+    },
+    "eslint.enable": true,
+    "eslint.format.enable": true,
+    "mochaExplorer.files": "test/{functional,unit}/**/*.test.js",
+    "mochaExplorer.ui": "test/tools/runner/metadata_ui.js",
+    "editor.formatOnSave": false,
+    "editor.rulers": [100],
+    "editor.renderWhitespace": "selection",
+    "files.trimTrailingWhitespace": true,
+    "files.trimFinalNewlines": true,
+    "files.insertFinalNewline": true,
+    "typescript.tsdk": "node_modules/typescript/lib",
+    // I leave the coverage extension disabled when not using it so I leave these commented
+    // but these settings are nice when it is enabled
+    // "coverage-gutters.showGutterCoverage": false,
+    // "coverage-gutters.showLineCoverage": true,
+  },
+  "launch": {
+    "configurations": [
+      {
+        // Sometimes I need to run mocha myself and not via the sidebar
+        // Here I can add custom args or env variables
+        "name": "run mocha",
+        "type": "pwa-node",
+        "request": "launch",
+        "program": "node_modules/.bin/mocha",
+        "args": ["test/unit", "test/functional"]
+      }
+    ],
+    "compounds": []
+  },
+  "tasks": {
+    "version": "2.0.0",
+    "tasks": [
+      {
+        // Here is an optional watcher task (`npm test` will also type check you changes):
+        // Since this is the default build task it can be started with cmd+shift+b
+        // There will be a wrench and screw icon
+        // on the bottom bar where you can quick check build issues
+        "label": "watch TS",
+        "command": "npx",
+        "type": "shell",
+        "args": ["tsc", "-w"],
+        "problemMatcher": "$tsc-watch",
+        "isBackground": true,
+        "group": {
+          "kind": "build",
+          "isDefault": true
+        }
+      }
+    ]
+  },
+  "extensions": {
+    "recommendations": [
+      "dbaeumer.vscode-eslint",
+      "hbenl.vscode-test-explorer",
+      "hbenl.vscode-mocha-test-adapter",
+      "ryanluker.vscode-coverage-gutters",
+      "github.vscode-pull-request-github",
+      "mongodb.mongodb-vscode"
+    ],
+    "unwantedRecommendations": ["esbenp.prettier-vscode"]
+  }
+}
+</pre>
+</details>
+
+If you use this file you will get our recommended extensions suggested to you. If not, we recommend picking up `dbaeumer.vscode-eslint` at least to make sure any additional code is following style recommendations.
+
+```jsonc
```

Comment:
* Can we make this a valid settings file? (Can we wrap the object in a `settings` key?)
* Can we add this in as a `\/` nested tab with the key minimalMongodbNodeDriver.code-workspace`?

It's not clear if I need both, pick and choose, why there's two json sections.

Reply:
Added a little more text to describe it, lmk how this looks now. I'm thinking if someone doesn't want to use the workspace file then they probably just want the few important settings in a quick copy paste. But I can do the `minimalMongodbNodeDriver.code-workspace` idea